notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb

```diff
@@ -82,11 +87,11 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services and resources:\n",
         "\n",
-        "- `Vertex AI Training` service\n",
-        "- `Vertex AI Prediction` service\n",
-        "- `Vertex AI Model Registry`\n",
-        "- `Vertex AI Model` resources\n",
-        "- `Vertex AI Endpoint` resources\n",
+        "- Vertex AI Training service\n",
+        "- Vertex AI Prediction service\n",
+        "- Vertex AI Model Registry\n",
+        "- Vertex AI Model resources\n",
+        "- Vertex AI Endpoint resources\n",
```

Comment:
Should be:
- Vertex AI endpoint resources

Reply:
pushed the change, but to confirm, should this actually be "Vertex AI Endpoint" itself as "Endpoint" is the name of a resource itself?